### PR TITLE
Error reporting fixed for checking status of `minio`

### PR DESCRIPTION
**Pull request by k0taperk0t**
## Change Overview

This PR fixes error reporting for checking status of `minio`. Currently, if `minio` is not available or broken we will get the same error:
> Please install MinIO using 'make install-minio' and try again.

Which is pretty weird, when you have installed `minio` and it works fine in local environment. With this change we will able to understand what's wrong with `minio`. In my case it reports this:

> Error: Kubernetes cluster unreachable: invalid configuration: [unable to read client-cert /Users/sergey.aksenov/.minikube/profiles/minikube/client.crt for minikube due to open /Users/sergey.aksenov/.minikube/profiles/minikube/client.crt: no such file or directory, unable to read client-key /Users/sergey.aksenov/.minikube/profiles/minikube/client.key for minikube due to open /Users/sergey.aksenov/.minikube/profiles/minikube/client.key: no such file or directory, unable to read certificate-authority /Users/sergey.aksenov/.minikube/ca.crt for minikube due to open /Users/sergey.aksenov/.minikube/ca.crt: no such file or directory]

PR series:
* https://github.com/kanisterio/kanister/pull/2376
* https://github.com/kanisterio/kanister/pull/2366
* https://github.com/kanisterio/kanister/pull/2377
* => https://github.com/kanisterio/kanister/pull/2378
* https://github.com/kanisterio/kanister/pull/2379
* https://github.com/kanisterio/kanister/pull/2365

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

### CODE REVIEW

**Comment by k0taperk0t** [comment]
I agree to the DCO for all the commits in this PR.

File: build/test.sh [hunk 1]
```diff
@@ -51,8 +51,8 @@ fi
 echo
 
 check_dependencies() {
-    # Check if minio is already deployed
-    if helm status minio -n minio > /dev/null 2>&1 ; then
+    # Check if minio is already deployed. We suppress only StdOut for watching error if check will fail.
+    if helm status minio -n minio 1> /dev/null ; then
```
Comment (viveksinghggits) [inline build/test.sh:55]: Just for my understanding previous command was ignoring the stdout as well as stderr which would case actual error to be ignored. But the new command just ignores the stdout to make sure that we catch the stderr right?

Comment (k0taperk0t) [reply build/test.sh:55]: That's right.

File: build/test.sh [hunk 2]
```diff
@@ -51,8 +51,8 @@ fi
 echo
 
 check_dependencies() {
-    # Check if minio is already deployed
-    if helm status minio -n minio > /dev/null 2>&1 ; then
+    # Check if minio is already deployed. We suppress only StdOut for watching error if check will fail.
```
Comment (viveksinghggits) [inline build/test.sh:54]:
```suggestion
    # Check if minio is already deployed. We suppress only `stdout` and not `stderr` to make sure we catch errors if `helm status` fails
```

If you think its better.

**Review by viveksinghggits — APPROVED** [review]

**Pull request merged**